pages/common/batch.md

```diff
@@ -0,0 +1,16 @@
+# batch
+
+> Execute commands at a later time when the system load levels permit.
+> Make sure the atd (or atrun) is running.
+
+- Execute a command from standard input:
+
+`echo "{{./make_db_backup.sh}}" | batch`
+
+- Execute commands from a given file:
+
+`batch -f {{path/to/file}}`
+
+- Execute commands from standard input (press CTRL+D when finished):
```

Comment:
Not sure why, this line wasn't changed in the last commit, but the inline comment has been hidden nevertheless. It mentioned the need to adjust the formatting of the keyboard shortcut, as was done in at.md.